src/main/kotlin/org/jetbrains/research/libsl/visitors/AutomatonResolver.kt

```diff
@@ -134,7 +135,17 @@ class AutomatonResolver(
         val name = ctx.nameWithType().name.asPeriodSeparatedString()
         val typeReference = processTypeIdentifier(ctx.nameWithType().type)
         val expressionVisitor = ExpressionVisitor(context)
-        val initValue = ctx.expression()?.let { right -> expressionVisitor.visitExpression(right) }
+        val initValue = ctx.assignmentRight()?.let { right ->
```

Comment:
I'd reuse code from `ExpressionVisitor` class here (so, this code will be simplified to `ctx.assignmentRight()?.let { expressionVisitor.visitAssignmentRight(it) }`)